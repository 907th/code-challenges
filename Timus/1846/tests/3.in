10
+ 24
+ 27
+ 9
+ 24
+ 12
- 24
- 24
- 12
- 27
- 9
