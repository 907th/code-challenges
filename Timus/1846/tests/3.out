24
3
3
3
3
3
3
9
9
1
